parent/pom.xml

```diff
@@ -59,6 +57,13 @@
         <yumArtifactName>emodb-web-${project.version}</yumArtifactName>
         <yumArtifactId>emodb-web</yumArtifactId>
 
+        <jacoco.version>0.8.6</jacoco.version>
+        <skipCC>true</skipCC>
+
+        <sonar.version>3.7.0.1746</sonar.version>
+        <sonar.sources>src/main</sonar.sources>
+        <sonar.test.sources>src/test</sonar.test.sources>
```

Comment:
Where is `sonar.host.url` set?

Reply:
it's been set in jenkins job, we do that to not expose BV internals